doc/workshop/exercises/analysis/dtm/dtm.rst

```diff
@@ -51,7 +51,7 @@ Pipeline breakdown
 1. Reader
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-``denoised-ground-only`` is the |LASzip| file we will clip. You should have
+``denoised-ground-only`` is the :ref:`COPC <copc.io>` file we will clip. You should have
```

Comment:
Does this reference actually resolve? If you want a link to COPC website, it would be 
```
`COPC <https://copc.io>`__
```